test/test_api/test_usages.py

```diff
@@ -3,7 +3,7 @@
 
 def test_import_references(Script):
     s = Script("from .. import foo", path="foo.py")
-    assert [usage.line for usage in s.get_references(line=1, column=18)] == [1]
```

Comment:
It's intentional.

(I just touched this, because there was an issue with the test when I refactored some things. After that it was not there anymore, but I still thought that the change made it more understandable - and functionally it's exactly the same).